samples/compression/lz4/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: lz4 sample
+  description: lz4 data compression library
+
+tests:
+  sample.compression.lz4:
+    platform_allow: nrf52840dk_nrf52840
+    platform_exclude: native_posix qemu_x86
+    harness: compression
```

Comment:
I don't think `harness: compression` makes any use. `harness` is rather used when you need something external, like network or keyboard. Here however compression is just something in software, so I don't think we need to specify anything here.